docs/drafts/design-system/color-mapping.md

```diff
@@ -0,0 +1,34 @@
+## Neutrals
+
+| Old         | New        |
+| ----------- | ---------- |
+| canvas-100  | neutral-1  |
+| canvas-200  | neutral-3  |
+| canvas-300  | neutral-4  |
+| canvas-400  | neutral-5  |
+| canvas-500  | neutral-6  |
```

Comment:
Hmm, not really clear. For some colors primary has `9` index, for some `4`. Is here some reason not to reserve `9` for all of them?

Reply:
It's because of the wrong decision we made in the past to separate canvas and neutral